chromadb/test/property/invariants.py

```diff
@@ -9,13 +12,38 @@ def count(api: API, collection_name: str, expected_count: int):
 
 
 def ann_accuracy(
-    api: API,
-    collection_name: str,
+    collection: Collection,
     embeddings: EmbeddingSet,
-    precision: float = 0.9,
-    recall: float = 0.9,
+    min_recall: float = 0.99,
 ):
-    """Validate that the API performs nearest_neighbor searches with the expected
-    precision and recall"""
-    # TODO: do in-process brute-force as comparison
-    pass
+    """Validate that the API performs nearest_neighbor searches correctly"""
+
+    # Validate that each embedding is its own nearest neighbor and adjust recall if not.
+    # Use k=10, because using k=1 with small N can affect recall very badly
```

Comment:
Confused by this comment, k=1 here?

Reply:
offline discussion between @atroyn and I - we were confused if this should ne n_results=10 or not - @levand wdyt?

Reply:
Thinking about this further, `n=10` doesn't work for most test cases. iirc the resolution here was .99 recall was acceptable for `n=1`